TLM/TLM/UI/ModUI.cs

```diff
@@ -179,23 +179,26 @@ public class ModUI : UICustomControl {
         }
 
         public static void EnableTool() {
-            Log._Debug("LoadingExtension.EnableTool: called");
+            Log._Debug("UIBase.EnableTool: called");
```

Comment:
It is not called UIBase anymore, that name was confusing. It is ModUI, i.e. root UI class for the entire TM:PE mod.